### ISSUE TITLE
unable to create a schema using a UNC file path

### ISSUE THREAD

**Issue by MaryMenges** [sissaschool/xmlschema#246]
Hi,

I'm unable to create a schema using a UNC file path. Local file paths work fine.
Example:
```
xsd_file_path = '\\host\share\file.xsd'
my_schema = xmlschema.XMLSchema11(xsd_file_path, defuse='always')
```
Error:
`<urlopen error [WinError 3] The system cannot find the path specified: '\\host\\share\\file.xsd'>`

It looks like the UNC path is not being normalized correctly. Are UNC paths supported? Is it possible to fix this?

**Comment by brunato** [comment]
Hi, 
sorry for delay in my response but supporting UNC paths involves a partial rewriting of the url normalization function. I will make this using, whenever possible, the support of pathlib library, so for example with an UNC path it will be:

```python
    url = url.strip()
    if url.startswith('\\'):
        return pathlib.PureWindowsPath(url).as_uri()  # UNC path
```

**Comment by brunato** [comment]
Fixed with release v1.6.3.
